core/src/main/java/com/testaem/aem/core/models/impl/package-info.java

```diff
@@ -0,0 +1,19 @@
+/*
```

Comment:
(cosmetic) Don't need Adobe copyright, this is not Adeobe's code)

Reply:
corrected in the last commit.